debian/changelog

```diff
@@ -1,3 +1,9 @@
+freerct (1:0.1~bzr5) UNRELEASED; urgency=medium
+
+  * Made everything work again after years of disuse.
```

Comment:
You also added several new things, would be nice if you mentioned those as well imho. (Doesn't need to be complete, but a few highlights, showing progress is being made rather than just update it to current software versions.

Reply:
I think the `debian/changelog` only refers to directly packaging-related changes, i.e. changes in the debian/ directory. All other changes should better get a separate ChangeLog file in the parent directory. But that's something for another branch, imho.